client/src/components/Card.js

```diff
@@ -1,24 +1,53 @@
 // @flow
 import React from 'react';
+import { connect } from 'react-redux';
+
 import Card from 'react-bootstrap/Card';
 import Badge from 'react-bootstrap/Badge';
 
+import { doGetPlace } from '../actions';
+
 type Props = {|
   id: string,
-  title: string,
+  name: string,
   description: string,
   address: string,
   score: number,
-  website: string
+  doGetPlace: (string) => void,
 |};
 
-export const ACard = ({ id, title, description, address, score, website }: Props) => (
-  <Card>
-    <Card.Body>
-      <Card.Title>{title} <Badge variant={score < 3 ? 'danger' : 'warning'}>{score}&#9733;</Badge></Card.Title>
-      <Card.Subtitle className="mb-2 text-muted">{address}</Card.Subtitle>
-      <Card.Text>{description}</Card.Text>
-      <Card.Link href={website} target="_blank">Visit website</Card.Link>
-    </Card.Body>
-  </Card>
-);
+class ACard extends React.Component<Props> {
```

Comment:
"a card" :)
because `Card` was already taken by `bootstrap`

Reply:
You can name it `Card` without being afraid, you always specify where to import the component from anyway. If you decide that you really want to give it a different name to avoid confusion, it's usually prefixed by your company/app - such as `WedoCard`.